sources.cfg

```diff
@@ -3,3 +3,9 @@ extends = http://kgs.4teamwork.ch/sources.cfg
 extensions = mr.developer
 
 auto-checkout =
+# https://github.com/collective/transmogrify.dexterity/pull/18
+  transmogrify.dexterity
+
+[branches]
+# https://github.com/collective/transmogrify.dexterity/pull/18
+transmogrify.dexterity = add-datetime-deserializer
```

Comment:
@mbaechtold could you reset all changes in sources.cfg?
I've released `transmogrify.dexterity = 1.6.0`...